__tests__/Unit/Components/Issues/Card.test.tsx

```diff
@@ -1,33 +1,85 @@
 import { render } from '@testing-library/react';
 import Card from '@/components/issues/Card';
+import MarkdownRenderer from '@/components/MarkdownRenderer/MarkdownRenderer';
 import { IssueItem } from '@/interfaces/issueItem.type';
 
-const DEFAULT_PROPS: IssueItem = {
-    html_url: 'https://github.com/Real-Dev-Squad/todo-action-items/issues/11',
+const issue_one: IssueItem = {
+    html_url: 'https://github.com/dummy-owner/dummy-repo/issues/11',
     id: 732825067,
-    title: 'Status view to all features being built for our app',
+    title: 'Dummy issue',
     user: {
-        login: 'ankushdharkar',
+        login: 'dummyuser',
+        html_url: 'https://github.com/dummyuser',
     },
     labels: [],
     state: 'open',
     assignee: {
-        login: 'whyDontI',
+        login: 'dummyassignee',
+        html_url: 'https://github.com/dummyassignee',
     },
     created_at: '2020-10-30T02:08:48Z',
-    body: 'We want to have a status page where we can view (and in future sort, filter) all the major tasks happening under development.\r\n\r\nThis will give us some visibility into what everyone is currently building and make it easier to collaborate.\r\n\r\nFirst view would look like this:\r\n![Roadmap](https://user-images.githubusercontent.com/1935403/97651644-ebf2e200-1a19-11eb-8a92-026d10ed8eee.png)\r\n',
+    body: null,
+};
+const issue_two: IssueItem = {
+    html_url: 'https://github.com/dummy-owner/dummy-repo/issues/11',
+    id: 732825067,
+    title: 'Dummy issue',
+    user: {
+        login: 'dummyuser',
+        html_url: 'https://github.com/dummyuser',
+    },
+    labels: [],
+    state: 'open',
+    assignee: {
+        login: 'dummyassignee',
+        html_url: 'https://github.com/dummyassignee',
+    },
+    created_at: '2020-10-30T02:08:48Z',
+    body: 'Dummy issue description',
```

Comment:
can we move these to a seperate file like fixtures/mocks?

Reply:
done